webpack/assets/javascripts/react_app/components/AuditsList/ActionLinks.js

```diff
@@ -0,0 +1,83 @@
+import $ from 'jquery';
+import React from 'react';
+import { Col } from 'patternfly-react';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { notify } from '../../../foreman_toast_notifications';
+
+const notifyOnError = (responseText) => {
+  try {
+    const response = JSON.parse(responseText);
+    notify({ message: `<p>${response.message}</p>`, type: 'error' });
+  } catch (ex) {
+    notify({ message: `<p>${responseText}</p>`, type: 'error' });
+  }
+};
+
+class ActionLinks extends React.Component {
+  constructData(dataInfo) {
+    const changedData = {};
+    const auditedChanges = Object.entries(this.props.auditedChanges);
+    if (auditedChanges.length) {
+      auditedChanges.forEach(([key, value]) => {
+        [changedData[key]] = value;
+      });
+    }
+    if (dataInfo.id) {
+      changedData.id = dataInfo.id;
+    }
+    return changedData;
+  }
+
+  handleClick(dataInfo, e) {
+    if (!$.rails.confirm(__('Do you really want to revert these changes?'))) {
```

Comment:
You can use `Confirm Dialog` [component](https://github.com/Katello/katello/blob/e8ff2f4ee53ac8bae98cb3c47c3d798eca90fd03/webpack/move_to_foreman/components/common/ConfirmDialog/ConfirmDialog.js)